app/gui2/shared/ast/tree.ts

```diff
@@ -1824,7 +1824,7 @@ applyMixins(MutableNumericLiteral, [MutableAst])
  *  GUI. We just need to represent them faithfully and create the simple cases. */
 type ArgumentDefinition<T extends TreeRefs = RawRefs> = (T['ast'] | T['token'])[]
 
-interface FunctionFields {
+export interface FunctionFields {
```

Comment:
I'm otherwise getting a type error in `GraphStore` definition, due to its return value using types that cannot be named. I'm not sure why this wasn't triggering in CI.